governance/votes/Executive vote - November 29, 2023.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Template - [Executive Vote] PSM-GUSD-A Removal from Cash Stablecoins, Coinbase Custody Debt Ceiling Increase, Various Parameter Changes, RWA Foundation Dao Resolution, Andromeda Legal Expenses, Vest Streams Updates and Spark Proxy-Spell - November 29, 2023
+summary: Setting PSM-GUSD-A Debt Ceiling to zero and removing from Autoline, Coinbase Custody (RWA014-A) Debt Ceiling increase, WBTC-A, WBTC-B, and WBTC-C Stability Fee changes, SBE parameter changes, RWA Foundation Dao Resolution, Andromeda Legal Expenses, Cease Vest Streams for Steakhouse, Set up Ecosystem Facilitator Vest Streams and TechOps and Spark proxy-spell
+date: 2023-11-29T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] PSM-GUSD-A removal from Cash Stablecoins, Coinbase Custody Debt Ceiling increase, various parameter changes, RWA Foundation Dao Resolution, Andromeda legal expenses, and Spark proxy-spell - November 29, 2023
+
+The Facilitators, Sidestream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+	
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+- PSM-GUSD-A will be removed from the Cash Stablecoin list by setting the [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) to zero and removing the vault from the [Debt Ceiling Instant Access (autoline)](https://manual.makerdao.com/module-index/module-dciam).
+- Coinbase Custody (RWA014-A) [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be increased to 1.5 billion.
+- WBTC-A, WBTC-B, and WBTC-C [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased by 0.07%.
+- The [hop (frequency)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-1) and [bump (lot size)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-3) SBE parameters will be changed to optimize for cost efficiency.
+- A RWA Foundation Dao Resolution will be approved.
+- A transfer will be made to cover Andromeda Legal expenses.
+- MKR and DAI Vest Streams for Steakhouse and TechOps will be Ceased.
```

Comment:
nit: cancel or yank would be preferable terms to cease. It also does not need to be capitalized.